bundler/lib/dependabot/bundler/update_checker/shared_bundler_helpers.rb

```diff
@@ -77,15 +77,18 @@ def in_a_temporary_bundler_context(error_handling: true)
           error_handling ? handle_bundler_errors(e) : raise
         end
 
-        # TODO: Use/remove argument
-        def in_a_native_bundler_context(_error_handling: true)
+        def in_a_native_bundler_context(error_handling: true)
           SharedHelpers.
             in_a_temporary_repo_directory(base_directory,
                                           repo_contents_path) do |_tmp_dir|
             write_temporary_dependency_files
 
             yield
           end
+        rescue Dependabot::SharedHelpers::HelperSubprocessFailed => e
+          # TODO: Add retrying
```

Comment:
This should probably implement the same retrying logic as the helper it is replacing, I just wanted to check the definition of retry-able errors still applied.

Reply:
I've got this mostly done locally so can merge my changes on top of this once it's merged